ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -3094,3 +3095,47 @@ def test_ipahealthcheck_mismatching_certificates_subject(self):
         finally:
             # cleanup
             tasks.uninstall_master(self.master)
+
+
+class TestIPAHealthcheckWithCALess(CALessBase):
+    """
+    Install CALess server with user provided certificate.
+    """
+    num_replicas = 0
+
+    def install(cls, mh):
```

Comment:
Pylint reports: 

```
************* Module ipatests.test_integration.test_ipahealthcheck
ipatests/test_integration/test_ipahealthcheck.py:3106: [E0213(no-self-argument), TestIPAHealthcheckWithCALess.install] Method 'install' should have "self" as first argument)
```
That’s because the method should be annotated with 

`@classmethod`